pkg/daemon/criruntime/imageruntime/interface.go

```diff
@@ -50,4 +50,5 @@ type ImagePullStatusReader interface {
 type ImageService interface {
 	PullImage(ctx context.Context, imageName, tag string, pullSecrets []v1.Secret, sandboxConfig *appsv1alpha1.SandboxConfig) (ImagePullStatusReader, error)
 	ListImages(ctx context.Context) ([]ImageInfo, error)
+	AuthInfos(ctx context.Context, imageName, tag string, pullSecrets []v1.Secret) []daemonutil.AuthInfo
```

Comment:
This method does not need to interact with the container runtime, so it can be defined directly into the utils to implement a public method.